src/Validator/Rules/OverlappingFieldsCanBeMerged.php

```diff
@@ -170,6 +172,7 @@ protected function getFieldsAndFragmentNames(
                 $fragmentNames
             );
 
+            // @phpstan-ignore-next-line PHPStan messes up the types of the passed-by-reference arguments
```

Comment:
maybe we should put these kind of issues into a baseline. _not sure whether we have not already discussed that with different conclusion_ 🤔

Reply:
makes sense here, done